viz/DepthMapVisualization.cpp

```diff
@@ -113,7 +124,7 @@ void DepthMapVisualization::updateMainNode ( osg::Node* node )
     }
     else
     {
-        colors->push_back(osg::Vec4(1.0,0,0.3,0.8));
+        colors->push_back(default_feature_color);
```

Comment:
I changed the default to the old color. That change was honestly merely for my convenience.